skimage/feature/_texture.pyx

```diff
@@ -264,3 +264,186 @@ def _local_binary_pattern(double[:, ::1] image,
                 output[r, c] = lbp
 
     return np.asarray(output)
+
+
+cdef inline cnp.double_t _integ(cnp.double_t[:, ::1] img,
+                                Py_ssize_t r0, Py_ssize_t c0,
+                                Py_ssize_t r1, Py_ssize_t c1) nogil:
+    """Integrate over the integral image in the given window
+
+    This method was created so that `multiblock_local_binary_pattern`
```

Comment:
No need to state this.  What needs to be stated is why the integrate method in _shared is not used.